dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/grid/data-storage.test.js

```diff
@@ -22,7 +25,253 @@ define([
             });
         });
 
-        describe('hasScopeChanged', function () {
+        describe('"initConfig" method', function () {
+
+            var model = new DataStorage({
+                dataScope: 'magento'
+            });
+
+            it('Check for defined ', function () {
+                expect(model.hasOwnProperty('initConfig')).toBeDefined();
+            });
+
+            it('Check method type', function () {
+                var type = typeof model.initConfig;
+
+                expect(type).toEqual('function');
+            });
+
+            it('Check returned value if method called without arguments', function () {
+                expect(model.initConfig()).toBeDefined();
+            });
+
+            it('Check returned value type if method called without arguments', function () {
```

Comment:
I don't think `Check for defined`, `Check method type`,  `Check returned value if method called without arguments` and `Check returned value type if method called without arguments` are necessary